src/ModuleList/Quiz/Quiz.tsx

```diff
@@ -1,75 +1,118 @@
 import React, { useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { SimpleGrid, Box, Flex } from '@chakra-ui/core';
-import { SubmitQuiz, CheckIcon, IncorrectIcon, QuestionLabel, QuizContainer, AnswerContainer } from './styled';
+import { SimpleGrid, Box, Flex, Text, useDisclosure } from '@chakra-ui/core';
+
+import { StringMap } from 'src/common/types';
 import { useQuizMeta } from 'src/common/api/curriculum/quiz';
 import { Question } from 'src/common/api/curriculum/quiz/requests';
-import { StringMap } from '../../common/types';
+import LoadingContainer from 'src/common/components/LoadingContainer';
+
+import {
+  SubmitQuiz,
+  CheckIcon,
+  IncorrectIcon,
+  QuestionLabel,
+  QuizContainer,
+  AnswerContainer,
+} from './styled';
+import QuizModal from './QuizModal';
 
 interface Params {
   quizId: string;
 }
 
 const Quiz = () => {
   const { quizId } = useParams<Params>();
+  const { data, isLoading } = useQuizMeta(Number(quizId));
+  const { isOpen, onClose, onOpen } = useDisclosure(false);
 
-  const { data } = useQuizMeta(Number(quizId));
+  const [isQuizSubmitted, setIsQuizSubmitted] = useState<boolean>(false);
+  const [questionIsCorrect, setQuestionIsCorrect] = useState<StringMap>({});
```

Comment:
In terms of variable names, shouldn't `questionIsCorrect` be `answerIsCorrect`? :)

Reply:
You're right!